product_supplierinfo_for_customer_sale/tests/test_product_name_search.py

```diff
@@ -58,3 +64,23 @@ def test_10_find_product_customer_code(self):
         self.assertEquals(len(product_names), 1)
         self.assertEquals(self.product.id, product_names[0][0])
         self.assertEquals('[code_product] Name_product', product_names[0][1])
+
+        # Create in product_1 a customer code similar to product, then
+        # name_search must find two products
+        self.assertFalse(self.product_1.customer_ids)
+
+        self.supplierinfo_dict.update({
+            'product_code': 'code_test_1',
```

Comment:
Use the `product.default_code` to return 2 records
One from supplier info and other one from product default search.